modules/101-cert-manager/openapi/config-values.yaml

```diff
@@ -101,3 +101,10 @@ properties:
     description: |
       The email used for sending notifications by LetsEncrypt.
     x-examples: ["example@example.com"]
+
+  certificateOwnerRef:
+    type: boolean
+    default: false
+    description: |
+      Whether to set the certificate resource as an owner of secret where the tls certificate is stored. When this flag is enabled, the secret will be automatically removed when the certificate resource is deleted.
```

Comment:
```suggestion
      Whether to set the certificate resource as an owner of a secret where the TLS certificate is stored. When this flag is enabled, the secret will be automatically removed when the certificate resource is deleted.
```

Reply:
Sorry but @nabokihms wrote the best text )